docs/developer/process/agent-release/pre-release.md

```diff
@@ -109,6 +109,12 @@ After all fixes have been cherry-picked:
 1. Push the changes to GitHub
 1. [Tag](#tag) with the appropriate `rc` number even if there were no changes
 
+After the RC build is done, manually run an [Agent Azure Pipeline](https://dev.azure.com/datadoghq/integrations-core/_build?definitionId=60) using the release branch, and the latest RC built. It will run all the e2e tests using a specific agent.
```

Comment:
```suggestion
After the RC build is done, manually run an [Agent Azure Pipeline](https://dev.azure.com/datadoghq/integrations-core/_build?definitionId=60) using the release branch, and the latest RC built. Select the options to run both Python 2 and Python 3 tests. This will run all the e2e tests against the current agent docker RCs. 
```
What do you think about this? Should python 2 and 3 always be run?

Reply:
At least python 3, but we can explicitly document to run both

Reply:
Sounds good!